lib/syskit/roby_app/plugin.rb

```diff
@@ -146,6 +148,53 @@ def self.setup(app)
 
                 rtt_core_model = app.default_loader.task_model_from_name("RTT::TaskContext")
                 Syskit::TaskContext.define_from_orogen(rtt_core_model, register: true)
+
+                # Log Transfer FTP Server spawned during Application#setup
+                tmp_root_ca = TmpRootCA.new
+
+                start_local_log_transfer_server(
+                    app.log_dir,
+                    tmp_root_ca.ca_user,
+                    tmp_root_ca.ca_password,
+                    tmp_root_ca.signed_cert
+                )
```

Comment:
Have to call stop_local_log_transfer_server in cleanup (need to define it)

~~~
def self.cleanup(app)
end
~~~